great_expectations/data_context/cloud_constants.py

```diff
@@ -0,0 +1,3 @@
+from typing_extensions import Final
```

Comment:
I think this is the first use of `Final` in our codebase.

But as you might guess.... they are mypy enforced constants.
https://mypy.readthedocs.io/en/stable/final_attrs.html?highlight=Final#final-names-methods-and-classes